internal/service/resources/collector/collector_acm.go

```diff
@@ -414,3 +449,110 @@ func (d *ACMDataSource) convertClusterToResourcePool(from data.Object) (to model
 
 	return
 }
+
+// makeCapacityInfo creates a map of strings of capacity attributes for the cluster
+func (d *ACMDataSource) makeCapacityInfo(cluster v1.ManagedCluster) map[string]string {
+	results := make(map[string]string)
+	tags := []string{"cpu", "ephemeral-storage", "hugepages-1Gi", "hugepages-2Mi", "memory", "pods"}
+	for _, tag := range tags {
+		if value, found := cluster.Status.Allocatable[v1.ResourceName(tag)]; found {
+			results[tag] = value.String()
+		}
+	}
+	return results
+}
+
+// getClusterClientConfig retrieves the cluster client config for the specified cluster.
+func (d *ACMDataSource) getClusterClientConfig(ctx context.Context, clusterName string) (*clientcmdapi.Config, error) {
+	// Fetch the kubeconfig for this cluster
+	kubeConfig, err := k8s.GetClusterKubeConfigFromSecret(ctx, d.hubClient, clusterName)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get cluster kubeconfig: %w", err)
+	}
+
+	config, err := clientcmd.Load(kubeConfig)
+	if err != nil {
+		return nil, fmt.Errorf("failed to unmarshal cluster config: %w", err)
+	}
+
+	return config, nil
+}
+
+// getDeploymentManagerExtensions builds the extensions required for the deployment manager
+func (d *ACMDataSource) getDeploymentManagerExtensions(ctx context.Context, clusterName string) (map[string]string, error) {
+	// Fetch the kubeconfig for this cluster and provide the info as extensions to the returned object. This
```

Comment:
Good idea to add it to the extensions from now

Reply:
Not my idea... that's how it was currently being done because of how it is defined in the conformance tests.  IMHO, the fact that the conformance tests are expecting specific things in extensions is wrong, but...

Reply:
yeah, weird